tests/test_nlu.yml

```diff
@@ -233,11 +233,7 @@ nlu:
     - what are you good at?
     - options
     - cool! can I do something else here?
-- intent: ask_when_next_event
-  examples: |
-    - what's the next rasa event
-    - is there a Rasa event in [San Francisco](location)
```

Comment:
Maybe we should add these examples to the other intent where they live now? :)

Reply:
Oh, totally. Adding it now 🙂